src/engine/strat_engine/thinpool/mdv.rs

```diff
@@ -128,31 +130,13 @@ impl MetadataVol {
         uuid: FilesystemUuid,
         fs: &StratFilesystem,
     ) -> StratisResult<()> {
-        let data = serde_json::to_string(&fs.record(name, uuid))?;
-        let path = self
-            .mount_pt
-            .join(FILESYSTEM_DIR)
-            .join(uuid.to_simple_ref().to_string())
-            .with_extension("json");
-
-        let temp_path = path.with_extension("temp");
-
         let _mount = MountedMDV::mount(self)?;
```

Comment:
Actually also related to this - the `MountedMDV` could hold an `openat::Dir` directory file descriptor for the mount point, which means it would more properly "own" the data and we wouldn't have to use the underscore `_` prefix to say "keep this alive even though it's not used".